outputs.tf

```diff
@@ -14,7 +14,42 @@
  * limitations under the License.
  */
 
-output "buckets_list" {
-  value       = "${compact(split(" ", data.external.bucket_retrieval.result["buckets_list"]))}"
-  description = "The buckets list within the created project"
+output "forseti-server-vm-name" {
+  description = "Forseti Server role vm name"
+  value       = "${google_compute_instance.forseti-server.name}"
+}
+
+output "forseti-server-vm-ip" {
+  description = "Forseti Server role ip address"
+  value       = "${google_compute_instance.forseti-server.network_interface.address}"
+}
+
+output "forseti-client-vm-name" {
+  description = "Forseti Client role vm name"
+  value       = "${google_compute_instance.forseti-client.name}"
+}
+
+output "forseti-client-vm-ip" {
+  description = "Forseti Cleint role vm name"
+  value       = "${google_compute_instance.forseti-client.network_interface.address}"
```

Comment:
```suggestion
  value       = "${google_compute_instance.forseti-client.network_interface.network_ip}"
```

Reply:
+1

Change to `value = "${google_compute_instance.forseti-client.network_interface.0.network_ip}"`